apps/meteor/app/meteor-accounts-saml/server/lib/settings.ts

```diff
@@ -117,9 +118,15 @@ export const loadSamlServiceProviders = async function (): Promise<void> {
 					const samlConfigs = getSamlConfigs(key);
 					SAMLUtils.log(key);
 					await LoginServiceConfiguration.createOrUpdateService(serviceName, samlConfigs);
+					void notifyOnLoginServiceConfigurationChangedByService(serviceName);
 					return configureSamlService(samlConfigs);
 				}
-				await LoginServiceConfiguration.removeService(serviceName);
+
+				const { deletedCount } = await LoginServiceConfiguration.removeService(serviceName);
+				if (deletedCount) {
+					void notifyOnLoginServiceConfigurationChangedByService(serviceName, 'removed');
```

Comment:
since `notifyOnLoginServiceConfigurationChangedByService` performs a `find`, this won't work since `serviceName` was just removed from DB..